server/graphql/v2/mutation/ExpenseMutations.ts

```diff
@@ -291,6 +292,10 @@ const expenseMutations = {
     async resolve(_: void, args, req: express.Request): Promise<typeof Expense> {
       checkRemoteUserCanUseExpenses(req);
 
+      if (args?.paymentParams?.twoFactorAuthenticatorCode) {
+        req.headers[TwoFactorAuthenticationHeader] = `totp ${args.paymentParams.twoFactorAuthenticatorCode}`;
+      }
+
```

Comment:
That's a good hack for now but I think we should open a followup issue about deprecating this `args?.paymentParams?.twoFactorAuthenticatorCode` in favor of the new header you've implemented.

Reply:
I will open an issue to deprecate and remove this from the frontend as well.